src/app/admin/components/learning-object-list-item/learning-object-list-item.component.ts

```diff
@@ -245,6 +247,15 @@ export class LearningObjectListItemComponent implements OnChanges {
     this.showHierarchyBuilder = val;
    }
 
+   releaseHierarchy() {
+    this.hierarchyService.releaseHierarchy(this.learningObject.id)
+    .then(() => {
+      location.reload();
```

Comment:
While this is happening, should there be an indicator that the hierarchy is being released? Loading icon?

Reply:
Yeah I can make put a loading icon popup, but I am hesitant to put a success message, because sometimes you will have to click 'Release Hierarchy' a few times because sometimes Hierarchy Service will only release children and not the full hierarchy.